tests/test_perf.py

```diff
@@ -152,3 +152,75 @@ def _astropy():
             benchmark(_astropy)
         else:
             benchmark(_nifty)
+
+
+@pytest.mark.parametrize('Nf', [1_000])
+@pytest.mark.parametrize('nterms', [2, 4])
```

Comment:
Right now, the test output is comparing nterms=2 and nterms=4 in the same table. I think there's not much point in comparing two different nterms values; let's just use 4 for now.